Assets/LeapMotion/Editor/CustomEditorBase.cs

```diff
@@ -118,6 +119,19 @@ public class CustomEditorBase : Editor {
       }
     }
 
+    /// <summary>
+    /// Defer rendering of a property until the end of the inspector.  Deferred properties
+    /// are drawn in the REVERSE order they are deferred!  NOT by the order they appear in the 
```

Comment:
OK, that sounds fine. Could you add a line in the summary like... "This way, deriving classes can have their deferred inspector fields rendered before their parent classes' deferred inspector fields"?